nmma_api/services/retrieval_queue.py

```diff
@@ -1,7 +1,7 @@
 import time
 from datetime import datetime
 
-from nmma_api.tools.expanse import retrieve, cancel_job
+from nmma_api.tools.expanse import retrieve, cancel_job, submit  # noqa 401
```

Comment:
I think we can remove my import of `submit` here along with the noqa comment, since we ended up not using the submit code in this service.